en/config/safety_simulation.md

```diff
@@ -0,0 +1,27 @@
+# Failsafe State Machine Simulation
+
+This page can be used to simulate the actions of the PX4 failsafe state machine under all possible configurations and conditions.
+
+The simulation runs the same code in the browser as is executed on the vehicle, in real-time (the simulation is automatically kept in sync with the latest version of the code).
+Note that any delayed action (`COM_FAIL_ACT_T`) will also be delayed in the simulation.
+
+To use it:
+
+1. First configure the parameters on the left.
```

Comment:
FYI I modified the list a bit, but not sure if it is better or not. I wanted to make clear that you must do the parameters and vehicle first, and that all other settings may impact each other - i.e. you might want to change conditions before or after you change things in the state.